pkg/controller/command/verifiable/command.go

```diff
@@ -62,20 +72,66 @@ const (
 	// error messages
 	errEmptyCredentialName = "credential name is mandatory"
 	errEmptyCredentialID   = "credential id is mandatory"
+	errEmptyDID            = "did is mandatory"
 
 	// log constants
 	vcID   = "vcID"
 	vcName = "vcName"
+
+	creatorParts = 2
+	// Ed25519VerificationKey supported Verification Key types
+	Ed25519VerificationKey = "Ed25519VerificationKey"
```

Comment:
We should probably switch type to JwsVerificationKey2020 and use that type within the system. Can be done later.

Reply:
will create todo

Reply:
Need to support both.

Reply:
@talwinder50 Any task for this? I don't see any TODO here

Reply:
JwsVerificationKey2020 type for verification #1639. Added todo now